src/dynamic-import/default/nested-if-braceless.template

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018 Rick Waldron. All rights reserved.
+// Copyright (C) 2018 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+path: language/module-code/dynamic-import/usage/nested-if-braceless-
+name: nested if syntax
+esid: sec-import-call-runtime-semantics-evaluation
+info: |
+  ImportCall :
+      import( AssignmentExpression )
+
+  1. Let referencingScriptOrModule be ! GetActiveScriptOrModule().
+  2. Assert: referencingScriptOrModule is a Script Record or Module Record (i.e. is not null).
+  3. Let argRef be the result of evaluating AssignmentExpression.
+  4. Let specifier be ? GetValue(argRef).
+  5. Let promiseCapability be ! NewPromiseCapability(%Promise%).
+  6. Let specifierString be ToString(specifier).
+  7. IfAbruptRejectPromise(specifierString, promiseCapability).
+  8. Perform ! HostImportModuleDynamically(referencingScriptOrModule, specifierString, promiseCapability).
+  9. Return promiseCapability.[[Promise]].
+
+
+features: [dynamic-import]
+flags: [async]
+---*/
+
+if (true) import(/*{ params }*/);
```

Comment:
*Note: Posting on a file not hidden by GitHub.*

☝️ Based on [INTERPRETING](https://github.com/tc39/test262/blob/377f07f502890284d1e89ecb6b66343464fe14c0/INTERPRETING.md) async flagged tests should print `'Test262:AsyncTestComplete'` or `'Test262:AsyncTestFailure: ' + reason` by way of [`doneprintHandle`](https://github.com/tc39/test262/blob/master/harness/doneprintHandle.js). Without print a complete or fail the interpretation can hit a test timeout and interpret it as a failure.